spring-integration-core/src/main/java/org/springframework/integration/aggregator/FluxAggregatorMessageHandler.java

```diff
@@ -62,18 +64,24 @@ public class FluxAggregatorMessageHandler extends AbstractMessageProducingHandle
 	private CorrelationStrategy correlationStrategy =
 			new HeaderAttributeCorrelationStrategy(IntegrationMessageHeaderAccessor.CORRELATION_ID);
 
+	@Nullable
 	private Predicate<Message<?>> boundaryTrigger;
 
-	private Function<Message<?>, Integer> windowSizeFunction = FluxAggregatorMessageHandler::sequenceSizeHeader;
+	@Nullable
+	public Function<Message<?>, Integer> windowSizeFunction = FluxAggregatorMessageHandler::sequenceSizeHeader;
```

Comment:
This is not correct. The `@Nullable` on the property type would mean that `FluxAggregatorMessageHandler::sequenceSizeHeader` lambda is null.
However that is not the case.
Here we are talking about result of `message.getHeaders().get(IntegrationMessageHeaderAccessor.SEQUENCE_SIZE, Integer.class)`.
Therefore it is `null` for function invocation.
Hence, has to be like this:
```
private Function<Message<?>, @Nullable Integer> windowSizeFunction = FluxAggregatorMessageHandler::sequenceSizeHeader;
```

I'm not sure why have you made this property as `public`, though...